platform/search/src/services/GenericSearchProvider.js

```diff
@@ -42,11 +44,12 @@ define([
      * @param {TopicService} topic the topic service.
      * @param {Array} ROOTS An array of object Ids to begin indexing.
      */
-    function GenericSearchProvider($q, $log, modelService, workerService, topic, ROOTS) {
+    function GenericSearchProvider($q, $log, modelService, workerService, topic, ROOTS, openmct) {
```

Comment:
The big list of dependencies here gives me a little pause. Are all of these still needed? It seems at least that `modelService` could be pruned, since all objects can be retrieved via the composition API.